dart-impl/base/src/hwinfo.c

```diff
@@ -354,6 +345,36 @@ dart_ret_t dart_hwinfo(
                  hw.num_cores, hw.core_id, hw.cpu_id);
 #endif /* DART_ENABLE_HWLOC */
 
+#ifdef DART_ENABLE_LIKWID
+  DART_LOG_TRACE("dart_hwinfo: using likwid");
+  /*
+   * see likwid API documentation:
+   * https://rrze-hpc.github.io/likwid/Doxygen/C-likwidAPI-code.html
+   */
+  int likwid_ret = topology_init();
+  if (likwid_ret < 0) {
+    DART_LOG_ERROR("dart_hwinfo: "
+                   "likwid: topology_init failed, returned %d", likwid_ret);
+  } else {
+    CpuInfo_t     info = get_cpuInfo();
+    CpuTopology_t topo = get_cpuTopology();
+    if (hw.min_cpu_mhz < 0 || hw.max_cpu_mhz < 0) {
+      hw.min_cpu_mhz = info->clock;
+      hw.max_cpu_mhz = info->clock;
+    }
+    if (hw.num_numa < 0) {
+      hw.num_numa    = hw.num_sockets;
```

Comment:
Is there a way to get `hw.num_sockets` from likwid? Otherwise both will be -1 if hwloc is not available

Reply:
I have checked and found how to get numa and socket numbers in likwid :) see my commit below